examples/Utils/b3Clock.cpp

```diff
@@ -143,8 +141,41 @@ unsigned long int b3Clock::getTimeMilliseconds()
 #endif
 }
 
-	/// Returns the time in us since the last call to reset or since 
-	/// the Clock was created.
+/// Gets the system time in milliseconds
+unsigned long int b3Clock::getSystemTimeMilliseconds() {
```

Comment:
This seems to be a duplicate of the already existing 'getTimeMilliseconds', why?